### DISCUSSION TITLE
Docker Compose + ESP Thread Border Router + Config

### DISCUSSION THREAD

**Discussion by martin8310** [openthread/openthread#9915] [General]
Hello everyone,

Can someone explain to me how I can get the combination (Docker Compose + ESP Thread Border Router) to work?

I bought the “ESP Thread Border Router” board and flashed it. The part can also be reached via the IP "http://192.168.1.22/index.html", but what happens next?

I now wanted to install OTBR Docker as Docker. But I'm missing as example for a yaml file. Does anyone have it?

regards
Martin

**Comment by chshu** [comment]
@martin8310 OTBR docker is not required to play with ESP Thread BR board, please follow the docs here: https://docs.espressif.com/projects/esp-thread-br.

- [OTBR Docker](https://openthread.io/guides/border-router/docker/run) runs on the Raspberry Pi or any other Linux-based machines, with a RCP to extend Thread/802.15.4 interface.
- ESP Thread BR board consists of two SoCs from Espressif: the host Wi-Fi SoC ESP32-S3 and RCP ESP32-H2. The ESP Thread BR SDK is built on top of esp-idf and openthread, not linux-based, so no docker compose.

You could try out the ESP Thread BR features by following the [codelabs](https://docs.espressif.com/projects/esp-thread-br/en/latest/codelab/index.html).
